M/MMG/build_tarballs.jl

```diff
@@ -0,0 +1,103 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "MMG"
+version = v"5.6.0"
+
+# Collection of sources required to build MMG
+sources = [
+    ArchiveSource("https://github.com/MmgTools/mmg/archive/refs/tags/v$(version).tar.gz",
+                  "bbf9163d65bc6e0f81dd3acc5a51e4a8c47a7fdae849abc26277e01154fe2437"),
+    ArchiveSource("https://github.com/ISCDtoolbox/Commons/archive/refs/heads/master.zip",
+                  "02fae58388a93692f3c73dcb4619bbcf838b479744cf6972edeca330f756125a"),
+    ArchiveSource("https://github.com/ISCDtoolbox/LinearElasticity/archive/refs/heads/master.zip",
+                  "b1898151fbc4294d05d1df2eade209f2e0e185e6e49f38b6fd36a1333dd591fd"),
```

Comment:
Please, **NEVER*** use snapshots of branches like this.  Branches are moving target, this recipe would become broken when they push to `master` again.  Also, why these are their own packages?

Reply:
Sorry about that, I wasn't sure since these dependencies aren't versioned very well. I've separated out LinearElasticity into it's own package now and changed the source to a particular commit. Note that now the pipelines for MMG will fail  for now because LinearElasticity is not registered.